enterprise/frontend/src/metabase-enterprise/caching/index.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { t, jt } from "ttag";
+import { PLUGIN_CACHING, PLUGIN_FORM_WIDGETS } from "metabase/plugins";
+import Link from "metabase/components/Link";
+import { CacheTTLField } from "./components/CacheTTLField";
+import { DatabaseCacheTTLField } from "./components/DatabaseCacheTTLField";
+import { QuestionCacheTTLField } from "./components/QuestionCacheTTLField";
+import {
+  getQuestionsImplicitCacheTTL,
+  validateCacheTTL,
+  normalizeCacheTTL,
+} from "./utils";
+
+PLUGIN_CACHING.getQuestionsImplicitCacheTTL = getQuestionsImplicitCacheTTL;
+
+PLUGIN_CACHING.cacheTTLFormField = {
+  name: "cache_ttl",
+  validate: validateCacheTTL,
+  normalize: normalizeCacheTTL,
+};
+
+function getDatabaseCacheTTLFieldDescription() {
+  return (
+    <span>
+      {jt`How long to keep question results. By default, Metabase will use the value you supply on the ${(
+        <Link
```

Comment:
`jt` is sneaky and converts this into an array, causing a missing `key` warning, so make sure you add a `key` prop to this `Link` component

Reply:
Thank you, fixed 👌